service/src/main/java/bio/terra/workspace/service/workspace/flight/RevokePetUsagePermissionStep.java

```diff
@@ -90,7 +93,7 @@ public StepResult undoStep(FlightContext context) throws InterruptedException {
       return StepResult.getStepResultSuccess();
     }
     petSaService.enablePetServiceAccountImpersonationWithEtag(
-        workspaceId, userAndPet, expectedEtag);
+        workspaceId, userAndPet, proxyGroupEmail, expectedEtag);
```

Comment:
Would it make sense to extend the UserWithPetSa class to carry the proxy group instead of modifying the interfaces all the way through?